src/components/FooterComponent/FooterImages/FooterImages.js

```diff
@@ -1,14 +1,19 @@
-const FooterImages = ({settings}) => {
-    const imageStyle = {
-        width: settings.width,
-        height: settings.height
-    }
+import React, { Component } from 'react';
+
+class FooterImages extends Component {
+    render() {
+        const { settings } = this.props;
+        const imageStyle = {
+            width: settings.width,
+            height: settings.height
+        };
 
-    return (
-        <div className="footer__data__lable__content__images__item">
-            <img src={settings.icon} alt="icons" className="footer__data__lable__content__images__item__img" style={imageStyle}/>
-        </div>
-    )
+        return (
+            <div className="footer__data__lable__content__images__item">
```

Comment:
Look you use BEM methodologic for styling, This component is a block, and their div should have a block name, i.e. ```footer-image``` and that's it. Class for the img tag in this case should be ```footer-image__image``` and this component should have it's own style file.